tests/test_suspend_resume/src/xc_ptr.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016-2021 XMOS LIMITED.
```

Comment:
If this is a new file, the copyright date should be just '2021'.

Reply:
Rm'ed file